tests/Tests/500__jupyterhub/jupyterhub-basic-auth.robot

```diff
@@ -0,0 +1,44 @@
+*** Settings ***
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          DebugLibrary
+
+#Suite Setup      Begin Web Test
+Suite Setup      Accelerated Setup Suite
+Suite Teardown   End Web Test
+
+
+*** Keywords ***
+Accelerated Setup Suite
+  Set Library Search Order  SeleniumLibrary
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+*** Variables ***
+
+*** Test Cases ***
+
+Open RHODS Dashboard
+  [Tags]  Start Sequence
+  Capture Page Screenshot
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
+  Capture Page Screenshot
+
+Can Launch Jupyterhub
+  [Tags]  Start Sequence
+  Capture Page Screenshot
+  Fix Spawner Status
+  Capture Page Screenshot
+  Launch JupyterHub From RHODS Dashboard Link
+  Capture Page Screenshot
+
+Can Login to Jupyterhub
+  [Tags]  Start Sequence
+  Capture Page Screenshot
+  Login To Jupyterhub  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  ${authorization_required} =  Is Service Account Authorization Required
+  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+  Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+  Capture Page Screenshot
```

Comment:
why do you need these test cases in a separated file? we already have same ones in the repository. Before duplicating them we should understand the reason behind and if there are alternatives. 

For example, if you need those test cases for perf testing, then we can add a specific tag to the already implemented ones and assess the possibility to do changes according your needs

Reply:
I'm afraid this was written a long time ago by Erwan. What we're trying to do is to stimulate a stampede at different stages:
* (1) being able to logon to the dashboard at scale
* (2): (1)+being able to logon to JH at scale
* (3): (2)+being able to launch the same notebook at scale
Please let me know if this is already achievable.

Reply:
After discussions with @tarukumar I found out a simpler way to do this, without having to split the test into multiple files, with the `--exclude` flag.

See this file:
https://github.com/openshift-psap/ods-ci/blob/a3c68899045958d29bfc32407a199573eaa8b9d9/tests/Tests/500__jupyterhub/test-jupyterlab-psap-simplenotebook.robot

which defiles the following tags: `RHODS`, `Jupyterhub` and `Notebook`.
Our idea is to study the large scale behavior with `--exclude JupyterhubANDNotebook`, or `--exclude Notebook` or (nothing excluded)